packages/services/src/view-service/transaction-planner/index.test.ts

```diff
@@ -77,47 +78,9 @@ describe('TransactionPlanner request handler', () => {
         compactBlockSpacePrice: 120n,
       }),
     );
+    mockIndexedDb.hasNativeAssetBalance?.mockResolvedValueOnce(true);
```

Comment:
The mock has a `hasNativeAssetBalance` method, but the actual implementation has a `hasStakingAssetBalance` method. I assume this is a typo/old method name, but it makes me wonder if the mock is actually doing anything?